src/OddJobs/Types.hs

```diff
@@ -35,8 +35,22 @@ import Control.Monad.Logger (LogLevel)
 -- @
 type TableName = PGS.Query
 
-pgEventName :: TableName -> Query
-pgEventName tname = "job_created_" <> tname
+-- | Specifies the table names used for job management.
+data TableNames = TableNames
```

Comment:
This seemed the obvious name, but if there are there any future features that might require more database configuration than just table names, it might be worthwhile to make this name more general now.